kustomize/data-plane/environment-properties.env

```diff
@@ -0,0 +1,3 @@
+EVENT_BRIDGE_FLEET_SHARD_INDEX_IMAGE
+EVENT_BRIDGE_SSO_URL
```

Comment:
Apart from local development, I don't think this will ever be anything other than sso.redhat.com if that helps to simply this.

Reply:
Thanks! This could be set as a default, then.